core/src/main/scala/scalaz/syntax/Foldable1Syntax.scala

```diff
@@ -25,8 +25,8 @@ final class Foldable1Ops[F[_],A] private[syntax](val self: F[A])(implicit val F:
   final def distinctE1(implicit A: Equal[A]): NonEmptyList[A] = F.distinctE1(self)
   final def intercalate1(a: A)(implicit A: Semigroup[A]): A = F.intercalate1(self, a)
   final def msuml1[G[_], B](implicit ev: A === G[B], G: Plus[G]): G[B] = F.msuml1(ev.subst[F](self))
-  final def asum1[G[_], B](implicit ev: A === G[B], G: Plus[G]): G[B] = F.asum1(ev.subst[F](self))
-  final def asumMap1[G[_], B](f: A => G[B])(implicit G: Plus[G]): G[B] = F.asumMap1(self)(f)(G)
+  final def psum1[G[_], B](implicit ev: A === G[B], G: Plus[G]): G[B] = F.psum1(ev.subst[F](self))
```

Comment:
so we're dropping the `asum` syntax?

Reply:
Ah, we should keep it.

Reply:
Is `asum1` a Haskell name? If not, we don't need to keep that.